src/Init/Init_ResetParameter.cpp

```diff
@@ -106,6 +108,30 @@ void Init_ResetParameter()
    }
 
 
+// hybrid velocity dt (empirically determined CFL condition)
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+   if ( DT__HYBRID_VELOCITY < 0.0 )
+   {
+      DT__HYBRID_VELOCITY = 1.00;
```

Comment:
We may need to reduce the default value. Check the discussion on [Slack](https://calab-ntu.slack.com/archives/C024EQQJ9DL/p1695012720469119?thread_ts=1694592378.577929&cid=C024EQQJ9DL).